panic when pull arm image

### What happen?

sealer report panic when pull the image on the server of arm architecture
 error :
```
[root@wfw-arm-01 ~]# ./sealer pull registry.cn-qingdao.aliyuncs.com/sealer-io/kubernetes:v1.21.12
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x7ec610]

goroutine 1 [running]:
github.com/alibaba/sealer/utils/platform.cpuVariant.func1()
	/home/runner/work/sealer/sealer/src/github.com/alibaba/sealer/utils/platform/cpu.go:52 +0xc0
sync.(*Once).doSlow(0x2cd23c0, 0x1b88aa0)
	/opt/hostedtoolcache/go/1.16.15/x64/src/sync/once.go:68 +0x11c
sync.(*Once).Do(...)
	/opt/hostedtoolcache/go/1.16.15/x64/src/sync/once.go:59
github.com/alibaba/sealer/utils/platform.cpuVariant(0x0, 0x180dc60)
	/home/runner/work/sealer/sealer/src/github.com/alibaba/sealer/utils/platform/cpu.go:45 +0x74
github.com/alibaba/sealer/utils/platform.GetDefaultPlatform(...)
	/home/runner/work/sealer/sealer/src/github.com/alibaba/sealer/utils/platform/platform.go:128
github.com/alibaba/sealer/utils/platform.GetPlatform(0x0, 0x0, 0x1d99a18, 0x40007a25c0, 0x4000050055, 0x5, 0x40009b1d38)
	/home/runner/work/sealer/sealer/src/github.com/alibaba/sealer/utils/platform/platform.go:137 +0x28
github.com/alibaba/sealer/sealer/cmd.glob..func15(0x2c74320, 0x40007a23d0, 0x1, 0x1, 0x0, 0x0)
	/home/runner/work/sealer/sealer/src/github.com/alibaba/sealer/sealer/cmd/pull.go:39 +0x6c
github.com/spf13/cobra.(*Command).execute(0x2c74320, 0x40007a23b0, 0x1, 0x1, 0x2c74320, 0x40007a23b0)
	/home/runner/work/sealer/sealer/src/github.com/alibaba/sealer/vendor/github.com/spf13/cobra/command.go:852 +0x320
github.com/spf13/cobra.(*Command).ExecuteC(0x2c74aa0, 0x20, 0x6, 0x7)
	/home/runner/work/sealer/sealer/src/github.com/alibaba/sealer/vendor/github.com/spf13/cobra/command.go:960 +0x268
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/work/sealer/sealer/src/github.com/alibaba/sealer/vendor/github.com/spf13/cobra/command.go:897
github.com/alibaba/sealer/sealer/cmd.Execute()
	/home/runner/work/sealer/sealer/src/github.com/alibaba/sealer/sealer/cmd/root.go:48 +0x30
main.main()
	/home/runner/work/sealer/sealer/src/github.com/alibaba/sealer/sealer/main.go:26 +0x30

```

### Relevant log output?

_No response_

### What you expected to happen?

_No response_

### How to reproduce it (as minimally and precisely as possible)?

_No response_

### Anything else we need to know?

_No response_

### What is the version of Sealer you using?

{"gitVersion":"v0.8.5","gitCommit":"f9c3d99","buildDate":"2022-04-28 14:18:32","goVersion":"go1.16.15","compiler":"gc","platform":"linux/arm64"}

### What is your OS environment?

NAME="Kylin Linux Advanced Server" VERSION="V10 (Tercel)" ID="kylin" VERSION_ID="V10" PRETTY_NAME="Kylin Linux Advanced Server V10 (Tercel)" ANSI_COLOR="0;31"

### What is the Kernel version?

Linux wfw-arm-01.novalocal 4.19.90-21.7.ky10.aarch64 #1 SMP Thu Dec 17 16:31:09 CST 2020 aarch64 aarch64 aarch64 GNU/Linux

### Other environment you want to tell us?

- Cloud provider or hardware configuration:
- Install tools:
- Others:

--- a/utils/platform/cpu.go
+++ b/utils/platform/cpu.go
@@ -48,7 +48,7 @@ func cpuVariant() string {
 				logrus.Error(err)
 			}
 			model, err := getCPUInfo("model name")
-			if !strings.Contains(err.Error(), ErrNotFound.Error()) {
+			if err != nil && !strings.Contains(err.Error(), ErrNotFound.Error()) {
 				logrus.Error(err)
 			}
 			cpuVariantValue = GetCPUVariantByInfo(runtime.GOOS, runtime.GOARCH, variant, model)

